node_src/ethoscope_node/test/test_mysql_backup.py

```diff
@@ -6,18 +6,18 @@
 import traceback
 import re
 
-def TestMySQLDump():
+def TestMySQLDumpCSV():
 
     try:
 
-        mirror = MySQLdbCSVWriter("/Users/phrfp/Software/anaconda2/envs/ethoscope/home/tmp",
+        mirror = MySQLdbCSVWriter(
                         remote_db_name="test_etho_db",
                         remote_host="localhost",
                         remote_pass="",
                         remote_user="root")
 
 
-        mirror.update_roi_tables()
+        mirror.write_roi_tables("/Users/phrfp/Software/anaconda2/envs/ethoscope/home/tmp", True,10, False)
```

Comment:
We probably want to avoid using hardcoded paths, this will fail on any other system.